_config.yml

```diff
@@ -26,6 +26,7 @@ github:
   links_url: links.html
   twitter_url: https://twitter.com/AnonyPla
   mastodon_url: https://mastodon.social/@anonymousplanet
+  tor_url: http://thgtoallkcxrdv37u6knsc3pumk6cq6lqmcqlw3j5vkmyahkxive4jyd.onion
```

Comment:
Now you need to add this to the defaults configuration in layouts, otherwise you're defining the `tor_url` variable and not using it anywhere. 

```HTML
        <a href="{{ site.github.links_url }}" class="btn_small">Links</a>
      {% endif %}
      {% if page.url != "/about.html" %}
        <a href="{{ site.github.about_url }}" class="btn_small">About</a>
      {% endif %}
        <a rel="me" href="{{ site.github.tor_url }}" target="_blank" class ="btn_small">Tor Mirror</a>
        </section>
    <h5>GPG Key Fingerprint: 7DFF D747 1FB7 6E2A 8ABB  BCDD D769 B374 9E93 3B8A / Minisign public key: <a href="{{ site.github.minisign_url }}" style="color:#ff4700">minisign.pub</a></h5>
    <noscript><p class="jswarning">JavaScript is required to toggle light mode. JavaScript is not used for any other purpose.</p></noscript>
      </div>
    </header>
    <div class="container">
      <section id="main_content">
        {{ content }}
      </section>
    </div>
  </body>
</html>
```

Reply:
![placement](https://i.postimg.cc/y8hyY4Tr/placement.png)